### ISSUE TITLE
KNN and Pretrained Models

### ISSUE THREAD

**Issue by Sreyan88** [roman-vygon/triplet_loss_kws#2]
Hi there,

Thank you for the great repo! Can I please know if you guys have also open sourced the `KNN code` you have used after triplet loss trained representations and the `pre-trained models on triplet loss`?

Thank You

**Comment by roman-vygon** [comment]
Hello @Sreyan88!
Thank you for your interest to this repository.
The kNN implementation can be found in [infer_train.py](https://github.com/roman-vygon/triplet_loss_kws/blob/master/infer_train.py), specifically the  FaissKNeighbors class. You can also find documentation on how to lower the memory footprint of the kNN model [here](https://github.com/facebookresearch/faiss/wiki/Lower-memory-footprint).
Pre-trained models on google speech commands are in the [/runs folder](https://github.com/roman-vygon/triplet_loss_kws/tree/master/runs).

**Comment by Sreyan88** [comment]
Hi @roman-vygon ,

Thank you for your response! I am actually not being able to find the /runs folder you mentioned here. Could you please check?

**Comment by roman-vygon** [comment]
@Sreyan88 fixed it, sorry
